common/floppy_config.go

```diff
@@ -38,7 +38,7 @@ func (c *FloppyConfig) Prepare(ctx *interpolate.Context) []error {
 	}
 
 	for _, path := range c.FloppyDirectories {
-		if strings.IndexAny(path, "*?[") >= 0 {
+		if strings.ContainsAny(path, "*?[") {
```

Comment:
we only support the latest version of go, but we wouldn't break BC without due consideration